gene/cli.py

```diff
@@ -102,18 +102,18 @@ def dump_database(output_directory: Path, db_url: str):
         click.get_current_context().exit(1)
 
 
-def _update_normalizers(
-    normalizers: Collection[SourceName], db: AbstractDatabase, update_merged: bool
+def _update_normalizer(
+    sources: Collection[SourceName], db: AbstractDatabase, update_merged: bool
 ) -> None:
     """Update selected normalizer sources.
 
-    :param normalizers: names of sources to update
+    :param source: names of sources to update
```

Comment:
sorry i missed this one
```suggestion
    :param sources: names of sources to update
```